docs/components/servo/gpio.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Configure a gpio Servo"
+linkTitle: "gpio"
+weight: 90
+type: "docs"
+description: "Configure a gpio servo."
+tags: ["servo", "components"]
+icon: "img/components/servo.png"
+# SME: Rand
+---
+
+Configure a `gpio` servo to integrate a hobby servo controlled by general-purpose input/output (GPIO) pins on a non-`pi` model [board](/components/board) into your robot:
+
+{{< tabs name="Configure a gpio Servo" >}}
+{{% tab name="Config Builder" %}}
+
+Navigate to the **config** tab of your robot's page in [the Viam app](https://app.viam.com).
+Click on the **Components** subtab and navigate to the **Create component** menu.
+Enter a name for your servo, select the type `servo`, and select the `gpio` model.
+
+Click **Create component**:
+
+![An example configuration for a gpio servo in the Viam app Config Builder.](../img/gpio-servo-ui-config.png)
+
+Edit and fill in the `"attributes"` JSON to align with your `board` `name` and GPIO wiring.
+
+{{% /tab %}}
+{{% tab name="JSON Template" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+    "components": [
+        {
+            "name": "<your-servo-name>",
+            "type": "servo",
+            "model": "gpio",
+            "attributes": {
+                "pin": "<your-pin-number>",
+                "board": "<your-board-nam>"
+            }
+        }, ... <insert your board component config>
+    }
+```
+
+{{% /tab %}}
+{{% tab name="JSON Example" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "components": [
+    {
+        "name": "your-servo-name",
+        "type": "servo",
+        "model": "gpio",
+        "attributes": {
+            "pin": "16",
+            "board": "your-board-name"
+        }
+    }, 
+    {
+        "name": "your-board-name",
+        "type": "board",
+        "model": "jetson"
```

Comment:
```suggestion
    {
        "name": "local",
        "type": "board",
        "model": "jetson"
    }, 
    {
        "name": "my_servo",
        "type": "servo",
        "model": "gpio",
        "attributes": {
            "pin": "16",
            "board": "local"
        }
```
or similar; rather than being things they fill in for theirs, I think this is an example of an actual one;
I think configuring the board before the servo makes sense too